include/curl/curl.h

```diff
@@ -1314,9 +1316,9 @@ typedef enum {
      Note that setting multiple bits may cause extra network round-trips. */
   CINIT(HTTPAUTH, LONG, 107),
 
-  /* Set the ssl context callback function, currently only for OpenSSL ssl_ctx
-     in second argument. The function must be matching the
-     curl_ssl_ctx_callback proto. */
+  /* Set the ssl context callback function, currently only for OpenSSL or
+     WolfSSL ssl_ctx or mbedTLS mbedtls_ssl_config in second argument.
+     The function must be matching the curl_ssl_ctx_callback proto. */
```

Comment:
I think the sentence reads better if `be matching` is replaced with `match` and `proto` is expanded to `prototype`.

Reply:
True, and worth changing while we're at it.